arbitrum-docs/node-running/reference/software-releases.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 'ArbOS software releases'
+sidebar_label: 'ArbOS software releases'
+author: dlee
+---
+
+:::info
+This page is meant to give a high level summary of node software releases, with particular emphasis on releases that bump the ArbOS version. To find information on all releases of Nitro, including the latest release, and accompanying changelogs, please visit the [Nitro Github Repository](https://github.com/OffchainLabs/nitro/releases).
+
+:::
+
+The name of the node software that is used to power Arbitrum is called Nitro. While Nitro releases can get shipped as frequently as multiple times per week, there are a few Nitro releases that carry an upgrade to [ArbOS](https://forum.arbitrum.foundation/t/arbitrum-arbos-upgrades/19695) - the Layer 2 EVM hypervisor that facilitiates the execution environment of an Arbitrum chain. Releases of ArbOS can be considered as Arbitrum's equivalent to a hard fork and are special because they could alter a nodes ability to produce valid Arbitrum blocks and are therefore **mandatory to pick up** as a validator of an Arbitrum One and Nova. For Arbitrum One and Nova, ArbOS version upgrades require that the Arbitrum DAO pass a governance proposal. [This is an example of a Nitro release that contains an ArbOS version bump, specifically to ArbOS 11](https://github.com/OffchainLabs/nitro/releases/tag/v2.2.0).
+
+ArbOS is an essential part of the Nitro stack. Visit [this page to learn more about Nitro's architecture](../../inside-arbitrum-nitro/inside-arbitrum-nitro.mdx) and go [here to learn more about ArbOS](https://forum.arbitrum.foundation/t/arbitrum-arbos-upgrades/19695).
+
+# Naming and numbering scheme
+ArbOS releases starting from 20 onwards will use the name of plantery moons in our solar system, descending in alphabetical order (i.e. the next ArbOS upgrade after ArbOS 20 "Atlas" will be a planetary moon that begins with the letter "B"). 
+
+The number used to denote each upgrade will increment by 10, starting from ArbOS 20 (i.e. the next ArbOS upgrade after ArbOS 20 will be ArbOS 30). This was done because there are teams who have customized their Orbit chain's [behavior](../../launch-orbit-chain/how-tos/customize-stf.mdx) or [precompiles](../../launch-orbit-chain/how-tos/customize-precompile.mdx) and who may wish to use ArbOS's naming schema between official ArbOS version bumps (e.g. ArbOS 11 could be the name of a customized version of ArbOS for a project's L3 Orbit chain).
```

Comment:
I don't think we want to use arbos 11 as an example since that was a real version before we adopted jumping by 10, using 12 or 21 would work better